content/docs/glossary/decentralized-exchange.mdx

```diff
@@ -63,7 +64,7 @@ Looking at the diagram, there are a few orientation things to notice and think a
 
 Some assets will have a very thin or nonexistent orderbook between them. That's fine: as discussed in greater detail below, paths of orders can facilitate exchange between two thinly traded assets.
 
-## Cross-asset payments
+## Cross-asset payments (in fx this is referred to as multi-leg transactions)
```

Comment:
This parenthetical should be added to the below paragraph, rather than the title.